business/business-impl/src/main/java/com/tolean/elab/business/impl/dictionary/DictionaryServiceImpl.java

```diff
@@ -67,9 +68,35 @@ public DictionaryViewDto updateDictionaryItem(String dictionaryCode, Long dictio
 
         DictionaryItem dictionaryItem = dictionary.getDictionaryItem(dictionaryItemId);
         dictionaryItemMapper.intoDictionaryItem(dictionaryItemUpdateDto, dictionaryItem);
+
+        if (!dictionaryItem.isActive()) {
+          dictionary.setDefaultValue(null);
+        }
+
         dictionary = dictionaryRepository.save(dictionary);
 
         return dictionaryMapper.toDictionaryViewDto(dictionary);
     }
 
+    @Override
+    public DictionaryViewDto updateDefaultValue(String code, DictionaryUpdateDto dictionaryUpdateDto) {
```

Comment:
- [ ] dictionaryUpdateDto nie może być null. Dodaj checkNotnull - jest to metoda publiczna. A w metodach publicznych unika się podawania argumentów które mogą być null.